staging/cse/windows/nvidiagpudriverfunc.ps1

```diff
@@ -0,0 +1,244 @@
+function Start-InstallGPUDriver {
+    param(
+        [Parameter(Mandatory = $true)]
+        [bool]$EnableInstall,
+        [Parameter(Mandatory = $true)]
+        [PSCustomObject]$DriverUrlConfig
+    )
+  
+    if (-not $EnableInstall) {
+        Write-ConsoleLog "ConfigGPUDriverIfNeeded is false. GPU driver installation skipped as per configuration."
```

Comment:
I think that we need to investigate the log issue and then use wirte-log instead of Write-ConsoleLog

Reply:
Write-Log uses Write-Output instead of Write-Host. It will write to the output. If i use Write-Output, the $RebootNeeded (return value captured from Start-InstallGPUDriver) will capture all the logs along with other output. It will cause 2 issues: 

1) The logs will not be printed unless we handle them specially.
Something like this:
`$RebootNeeded = Start-InstallGPUDriver -EnableInstall $global:ConfigGPUDriverIfNeeded -DriverUrlConfig $DriverUrlConfig | Tee-Object -Variable "LogMessages"
`
2) The return value of Start-InstallGPUDriver will be more than a bool, which caused strange behavior in my standalone test.  (i.e. Start-InstallGPUDriver return false. But the return value also includes log message. Hence, the $RebootNeeded is interpreted as true, which causes misbehaviors.)

Reply:
Since Write-Log is used frequently in many helper functions, to avoid the same issue in the future, I will make code changes to accommodate the use of Write-Log.